lib/android-helpers.js

```diff
@@ -13,6 +13,7 @@ import {
 } from './unlock-helpers';
 import { EOL } from 'os';
 import semver from 'semver';
+import { validateCaps } from 'appium-base-driver/build/lib/basedriver/capabilities';
```

Comment:
this does not look like a valid import to me. The function should be present in package's index in order to be imported